package.json

```diff
@@ -6,21 +6,25 @@
   "scripts": {
     "cleanup": "shx rm -rf dist/",
     "test": "yarn build && jest",
-    "test:all": "yarn test && yarn test:env && yarn test:playgrounds",
+    "test:all": "yarn test && yarn test:env && yarn test:playgrounds && yarn test:e2e",
```

Comment:
Do you add the `yarn test:e2e` of angular because you plan to add other e2e tests? Otherwise this is only a test that should be in `test:playgrounds` instead of `test:all`, right?

Reply:
Playground tests are basically "do the playground build". Where as the `e2e` tests really tests `instant-meilisearch` the same way we test all meilisearch SDK's. 

Also, yes, i would like in the long run to have e2e tests for instantsearch envs: react, vue, js.